Resources/Prototypes/_SpaceCats/Entities/Clothing/Eyes/glasses.yml

```
@@ -0,0 +1,24 @@
+    protectionTime: 5
+  - type: ShowHealthBars
+  - type: Tag
+    tags:
+    - HamsterWearable
+    - WhitelistChameleon
+  - type: GuideHelp
+    guides:
+    - Security
+    - Antagonists
+  - type: IdentityBlocker
+    coverage: EYES
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)